content/08.methods.md

```diff
@@ -30,6 +30,16 @@ The ability to irrefutably prove manuscript existence at a past time could be im
 
 ### Author contributions
 
+#### Version 2.0
+
+We continued using the open repository on the GitHub version control platform ([`greenelab/deep-review`](https://github.com/greenelab/deep-review)) [@url:https://github.com/greenelab/deep-review], which was established to write the Version 1.0 manuscript.
+Drafted multiple sub-sections along with extensive editing, pull request reviews, or discussion:
```

Comment:
I'd place some placeholder here like "_pending_" to make it clear the empty fields indicate a work in progress and not an error.

Reply:
I still recommend making this change.  Having no text after the colon feels incomplete.

Reply:
Now generated automatically. Leaves out contribution categories with no contributions.